### PR TITLE
fix(GiniCaptureSDK): Fix color customisation of the skip button on the onboarding top navigation bar

### PR DESCRIPTION
**Pull request by a-szotyori**

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift [hunk 1]
```diff
@@ -16,12 +16,8 @@ class OnboardingViewController: UIViewController {
     private var navigationBarBottomAdapter: OnboardingNavigationBarBottomAdapter?
     private var bottomNavigationBar: UIView?
 
-    lazy var skipButton = UIBarButtonItem(title: NSLocalizedStringPreferredFormat(
-        "ginicapture.onboarding.skip",
-        comment: "Skip button"),
-                                          style: .plain,
-                              target: self,
-                              action: #selector(close))
+    lazy var skipButton = GiniBarButton(ofType: .skip)
```
Comment (ValentinaIancu-Gini) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift:19]: [suggestion] This property can be private.

**Review by ValentinaIancu-Gini — COMMENTED** [review]
@a-szotyori Thank you for helping us!

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift (hunk 1 above)
Comment (a-szotyori) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift]: Thank you, I made it private.

**Review by zladzeyka — APPROVED** [review]
thank you @a-szotyori

**Review by ValentinaIancu-Gini — APPROVED** [review]
@a-szotyori Thank you! Great job!

**Pull request merged**